client/src/components/application/Examine/CompName.vue

```diff
@@ -363,6 +363,7 @@
       },
       nameAccept() {
         this.$store.commit('decision_made', 'APPROVED');
+        this.$store.commit('currentCondition', null);
```

Comment:
nice! can you also include this in the nameReject(), so it clears out in that scenario as well - there is a minimal edge case where it is logical to reference conditions so it does make sense, but also we don't prevent the user from having conditions selected even when rejecting (even if it doesn't make sense) so might as well put it in for good measure.

Reply:
ah yes, thanks!